packages/flutter/lib/src/material/checkbox.dart

```diff
@@ -213,18 +214,43 @@ class Checkbox extends StatefulWidget {
 
   /// The color for the checkbox's [Material] when it has the input focus.
   ///
+  /// If [overlayColor] returns a non-null color in the [MaterialState.focused]
```

Comment:
the existing comment below is clearer, could you update it to mention that [overlayColor] is used first?

Reply:
See my comment below for the reasoning.